Documentation/Configuration/TypoScript/Syntax/FileImports/Index.rst

```diff
@@ -6,9 +6,10 @@
 File imports
 ============
 
-..  deprecated:: 13.4
+..  versionchanged:: 14.0
     The old school :typoscript:`<INCLUDE_TYPOSCRIPT:` syntax has been deprecated
-    and will be removed with TYPO3 v14.0. See :ref:`typoscript-syntax-includes-migration`.
+    with version 13.4 and removed with TYPO3 v14.0.
```

Comment:
```suggestion
    with version 13.4 and was removed with TYPO3 v14.0.
```

Reply:
Actually, the old version was correct, as "has been" refers to both versions IMHO.